src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -53,8 +58,10 @@ public IActionResult Status()
             {
                 Version = this.fullNode.Version?.ToString() ?? "0",
                 Agent = this.nodeSettings.Agent,
+                PID = Process.GetCurrentProcess().Id,
```

Comment:
We tent to have meaningful names for variables so probably something pike `ProcessId` will be better here?